config/base/env/config

```diff
@@ -3,6 +3,7 @@ DB_PASSWORD=
 DB_HOST=
 DB_PORT=5432
 DB_NAME=
+DB_SCHEMA=public
```

Comment:
This becomes the default "table prefix" value in our distribution. Will this break the database initialization/auto-migration phase of the deployment on upgrade? Is there risk of data being "orphaned" because the apiserver will create a whole new schema in Postgres?